EIPS/eip-7007.md

```diff
@@ -0,0 +1,160 @@
+---
+eip: 7007
+title: zkML AIGC-NFTs
+description: An ERC-721 extension interface for zkML based AIGC-NFTs.
+author: Cathie So (@socathie), Xiaohang Yu <xhyumiracle@hyperoracle.io>, Huaizhe Xu (@HuaizheXu), Kartin <kartin@hyperoracle.io>
```

Comment:
You'll want to add your GitHub username as an author or else you won't be able to edit the proposal without getting approvals from the other authors.